it/test/utils/IntegrationTest.scala

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils
+
+import com.github.tomakehurst.wiremock.client.WireMock.{aResponse, get, stubFor, urlMatching}
+import com.github.tomakehurst.wiremock.stubbing.StubMapping
+import config.FrontendAppConfig
+import handlers.ErrorHandler
+import models.User
+import org.scalatest.OptionValues
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpecLike
+import org.scalatestplus.play.guice.GuiceOneServerPerSuite
+import play.api.http.HeaderNames
+import play.api.i18n.{Lang, Messages, MessagesApi}
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.libs.ws.WSClient
+import play.api.mvc.MessagesControllerComponents
+import play.api.test.FakeRequest
+import play.api.{Application, Environment, Mode}
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.http.test.WireMockSupport
+
+import java.time.LocalDate
+import java.util.UUID
+import scala.concurrent.duration.Duration
+import scala.concurrent.{Await, Awaitable, ExecutionContext}
+
+trait IntegrationTest extends AnyWordSpecLike with Matchers with GuiceOneServerPerSuite with WireMockSupport with OptionValues {
```

Comment:
This trait bring in a lot that probably isn't needed. Can you review and only add what's required

Reply:
Was just `OptionValues` that wasn't required